tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -370,22 +373,22 @@ def test_volume_data_assistant_plot_descriptive(
         "data_asset_name": "my_reports",
     }
 
-    data_assistant: DataAssistant = VolumeDataAssistant(
+    data_assistant: VolumeDataAssistant = VolumeDataAssistant(
```

Comment:
@NathanFarmer Please revert back to `DataAssistant` (polymorphism must work; otherwise, we are still not out of the woods!).  Thanks!

Reply:
✅